src/Provider/Provider.Web/Controllers/VacanciesController.cs

```diff
@@ -38,5 +39,23 @@ public async Task<IActionResult> Vacancies([FromQuery] string filter, [FromQuery
 
             return View(vm);
         }
+
+        private void SetFilters(string filter, string search)
+        {
+            var value = $"filter:{filter}||search:{search}";
```

Comment:
Maybe use a class and Json serialize/desearlize into the cookie so we don't have to do any string manipulation.